src/core/crypto/secp256k1.ts

```diff
@@ -250,10 +251,21 @@ export class Secp256k1PrivateKey extends Serializable implements PrivateKey {
    *
    * @returns string representation of the private key
    */
-  toString(): string {
+  toHexString(): string {
     return this.key.toString();
   }
 
+  /**
+   * Get the private key as a AIP-80 compliant hex string.
+   *
+   * [Read about AIP-80](https://github.com/aptos-foundation/AIPs/blob/main/aips/aip-80.md)
+   *
+   * @returns AIP-80 compliant string representation of the private key.
+   */
+  toString(): string {
+    return PrivateKey.formatPrivateKey(this.key.toString(), "secp256k1");
```

Comment:
if now I am getting the AIP-80 compliant string, does it require me to use the ts-sdk when doing operations with this string? like when submitting a transactions?